### PR TITLE
Add new company: Renaissance

### PR DESCRIPTION
**Pull request by dhruvchandak30**
This is a modified version of, and should adhere to, the [Contributing Guidelines](../../main/.github/CONTRIBUTING.md).

* [ ] This PR contains housekeeping only (URL edits, copy changes etc) - **if this is checked, delete other lines that don't apply**
* [ ] You are an employee of the company mentioned and confirm all included details are correct.
* [x] You know your alphabet - company is listed in alphabetical order in the README.md file.
* [x] The company directly hires employees. ** No bootcamps / freelance sites / etc**
* [x] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such.
* [x] A [company profile](/company-profiles/example.md) is included - This is __required__ for new additions.
* [x] An entry has been added or edited within the [readme.md file](../../main/README.md).
* [x] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
* [x] __Region__ details the geographic regions in which this company employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
* [x] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

### CODE REVIEW

File: company-profiles/renaissance.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+
+## Remote and Location Info
+
+* Headquartered in the US - Wisconsin Rapids, WI
+* [Corporate Office Locations List](https://www.renaissance.com/about-us/contact-us/#office-locations)
+
+## Open Jobs and Applying
+
+[Click Here to apply](https://careers.smartrecruiters.com/Renaissance)
```
Comment (KieranRobson) [inline company-profiles/renaissance.md:28]: The layout and headings are standardised, see your other PR for correct layout #1687

**Review by KieranRobson — CHANGES_REQUESTED** [review]
Changes to be made.

**Comment by KieranRobson** [comment]
Thanks for the speedy changes, let me know when it is ready for review!

**Comment by dhruvchandak30** [comment]
> Thanks for the speedy changes, let me know when it is ready for review!

@KieranRobson  It is ready for review.

File: company-profiles/renaissance.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Renaissance 
+
+## Company blurb
+
+Renaissance’s solutions help educators analyze, customize, and plan personalized learning paths for students, allowing time for what matters—creating energizing learning experiences in the classroom. More than one-third of U.S. schools use Renaissance solutions to accelerate learning for millions of students.
+
+## Company technologies
```
[the remark this answers is not in the document]
Comment (dhruvchandak30) [reply company-profiles/renaissance.md:7]: It is nowhere mention which Technologies they use. Should I remove this section?

[the remark this answers is not in the document]
Comment (dougaitken) [reply company-profiles/renaissance.md:7]: Not ideal but keep the header for the section and leave it blank. The company is aimed at technology but it's not even clear on their job listings what technologies they use.

File: README.md [hunk 3]
```diff
@@ -593,6 +593,7 @@ Name | Website | Region
 [Remote Garage](/company-profiles/remote-garage.md) | http://www.remotegarage.club/ | India
 [RemoteBase](/company-profiles/remotebase.md) | https://remotebase.com | Worldwide
 [RemoteMore](/company-profiles/remotemore.md) | https://remotemore.com | WorldWide
+[Renaissance](/company-profiles/renaissance.md) | https://www.renaissance.com/ | United States, UK
```
Comment (dougaitken) [inline README.md:596]: I don't see anything on the site about UK so let's remove that and keep it as `United States`

**Review by dougaitken — CHANGES_REQUESTED** [review]
change the region in the readme line removing UK, and delete the content but keep the header for `Company technologies` @dhruvchandak30

**Review by dougaitken — APPROVED** [review]

**Comment by dougaitken** [comment]
@dhruvchandak30 Sorry for the delay. If you can, can you remove the Company technologies header? The build is complaining and I forgot a header needs to be removed if it's empty.

**Comment by dhruvchandak30** [comment]
> @dhruvchandak30 Sorry for the delay. If you can, can you remove the Company technologies header? The build is complaining and I forgot a header needs to be removed if it's empty.

Sure, I will remove it.

**Comment by dhruvchandak30** [comment]
@dougaitken  I have removed the Technologies' Header.

**Review by dougaitken — APPROVED** [review]

**Pull request merged**